demos/common/processing/src/config_factory.cpp

```diff
@@ -0,0 +1,120 @@
+/*
+// Copyright (C) 2018-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+#include "config_factory.h"
+
+#include <set>
+
+#include <samples/args_helper.hpp>
+#include <samples/common.hpp>
+#include <cldnn/cldnn_config.hpp>
+
+DECLARE_string(d);
+DECLARE_bool(pc);
+DECLARE_string(c);
+DECLARE_string(l);
+DECLARE_uint32(nireq);
+DECLARE_uint32(nthreads);
+DECLARE_string(nstreams);
+
+using namespace InferenceEngine;
+
+CnnConfig ConfigFactory::getUserConfig(const std::string& d, const std::string& l, const std::string& c, bool pc, uint32_t nireq, const std::string& nstreams, uint32_t nthreads) {
```

Comment:
Please use "Match whole word" option. Variables are named according to command line options (as we agreed, we pass command line options as parameters instead of using FLAGS_ macros)

Reply:
> Please use "Match whole word" option.

Web browsers may not have it.

> Variables are named according to command line options (as we agreed, we pass command line options as parameters instead of using FLAGS_ macros)

That actually means you are not forced now to match flags exactly in naming.

Reply:
Ping

Reply:
As I wrote, variables are named according to command line flags. There's no mistake in it

Reply:
I've modified names to avoid blocking of PR merge - currently it's more important than avoiding useless code changes.